### ISSUE TITLE
[new site support] https://ncap.org.uk/

### ISSUE THREAD

**Issue by Mightyden** [lovasoa/dezoomify#370]
https://ncap.org.uk/

NCAP is one of the largest collections of aerial imagery in the world, containing tens of millions of aerial images featuring historic events and places around the world.

https://ncap.org.uk/frame/24-1-20-26-8
https://ncap.org.uk/sites/default/files/frames/download/000/000/027/NCAP-000-000-027-678.jpg

### Current error message
Uncaught Error: Unable to fetch https://ncap.org.uk/sites/default/files/frames/download/000/000/027/NCAP-000-000-027-678.jpg/ImageProperties.xml

**Comment by lovasoa** [comment]
Hello,
  The images from this site are not accessible without being logged in. We have a wiki page about that case: https://github.com/lovasoa/dezoomify/wiki/How-to-access-protected-pages
  Do not hesitate to ask for more information here, or to contribute to the wiki yourself if it's not clear.

**Comment by Mightyden** [comment]
Okay Thx....

**Comment by Mightyden** [comment]
Hello,
Do you have an another solutuion?

**Comment by lovasoa** [comment]
I'm sorry, but I cannot provide further help. The site looks completely broken on my browser : 

![image](https://user-images.githubusercontent.com/552629/77856486-d1445a80-71f7-11ea-8bab-64ce8ad647b1.png)


And the URL you gave (https://ncap.org.uk/sites/default/files/frames/download/000/000/027/NCAP-000-000-027-678.jpg/ImageProperties.xml)  does not seem to exist

![image](https://user-images.githubusercontent.com/552629/77856507-fb961800-71f7-11ea-989b-31361e141e78.png)

**Comment by lovasoa** [comment]
I connected to the site with the credentials you sent, @Mightyden . I can see the images. The website doesn't provide ImageProperties.xml files and instead includes the image metadata directly in the page's source, which is currently not supported by dezoomify.

If you want to add support to it, your contribution will be welcome.

**Comment by lovasoa** [comment]
Let's leave this issue opened, since the site still doesn't work with dezoomify

**Comment by Mightyden** [comment]
@ lovasova: Have you now found a solution?

**Comment by lovasoa** [comment]
No, but I haven't looked for one either :) As I said, if you want to add support to it, your contribution will be welcome.

**Comment by a01020304** [comment]
There is a demo page on the site that allows zoomable.
Maybe staff can figire out link for zoomify for those and then once you purchase subscription links can be easily found.
https://ncap.org.uk/frame/1-1-35-1-110?pos=0

**Comment by Dr-Mx** [comment]
The site works for me when using the 'generic dezoomer': https://dezoomify.ophir.dev/#https://ncap.org.uk/sites/all/libraries/zoomify512/000/000/018/NCAP-000-000-018-209/TileGroup0/3-{{X}}-{{Y}}.jpg
You can find the filename from any thumbnail - https://ncap.org.uk/sites/default/files/frames/download/000/000/018/NCAP-000-000-018-209.jpg

**Comment by maximusdes1981** [comment]
https://github.com/jedrus2000/ncapimg   - I found this a long time ago. Everything worked. But now for some reason it doesn’t work. Perhaps because I work through a VPN. Please check it yourself.

**Comment by Dr-Mx** [comment]
> https://github.com/jedrus2000/ncapimg - I found this a long time ago. Everything worked. But now for some reason it doesn’t work. Perhaps because I work through a VPN. Please check it yourself.

You're right. Looks they've updated their system so you now need a unique 64 character hash for each image.

This is an example of the new url:
`https://ncap.org.uk/sites/all/libraries/zoomify.images/<HASH>/NCAP-000-000-000-000/TileGroup0/0-0-0.jpg`

It might be possible to reverse engineer this hash knowing the previous url parts, but I don't currently have a resolution for you.

**Comment by maximusdes1981** [comment]
Does anyone know if this approach still works? I cant find the szi URL.
https://github.com/lovasoa/dezoomify-rs/discussions/220

**Comment by sergey8888882** [comment]
I'm also having the same failure to get the zome in chunks due to the url update
